NickvisionMoney.Shared/Models/Account.cs

```diff
@@ -815,13 +817,19 @@ public async Task<bool> DeleteGroupAsync(uint id)
                 if (pair.Value.GroupId == id)
                 {
                     pair.Value.GroupId = -1;
+                    if (pair.Value.UseGroupColor)
+                    {
+                        pair.Value.UseGroupColor = false;
+                        pair.Value.RGBA = defaultTransactionColor;
```

Comment:
do we really need to change RGBA to default here? At least in GNOME RGBA is always saved from ColorDialogButton, even when UseGroupColor is true. It also can be other than the default value if a user changed it, so I think it does make sense to use it without changing to default.

Reply:
Yeah you are right, i'll remove that line.